src/components/MyProfile/ProfileMain/ProfileMain.tsx

```diff
@@ -41,9 +47,18 @@ const ProfileMain = async ({ profile }: { profile: ProfileModel }) => {
               </a>
             </li>
           )}
-          <li className={styles.copyEmail}>
-            <CopyEmail email="test@nerdbord.io" />
-          </li>
+          {isConnectedToNerdbord && (
+            <li className={styles.socialItem}>
+              <a
+                className={styles.socialLink}
+                href={'#'}
```

Comment:
**question:** This href is to implement in the next task, right?

Reply:
Yes, it will implement in the next task